### ISSUE TITLE
Can't register with FDev auth

### ISSUE THREAD

**Issue by InnerBushman** [jixxed/ed-odyssey-materials-helper#410]
When I click the Connect commander...  button in the app, the auth page is opened in my default browser. I enter my credentials and click login and nothing happens.
Does anyone else have problems with that? Are there any things I could check or try to make it work?

Other apps did not have problems with registering so I'm not sure what's wrong.

**Comment by jixxed** [comment]
what browser?

**Comment by InnerBushman** [comment]
Firefox 129.0.1 (64-bit)

**Comment by InnerBushman** [comment]
The question of which browser I'm using gave me some ideas to check so I started with other browser, in this case chromium.
By doing that I've noticed the browser is trying to open the URL with the handler program, but that doesn't work, so I started digging and found this thread:
https://github.com/jixxed/ed-odyssey-materials-helper/issues/191
Right now I'm able to run the app from the desktop file but the API integration is not yet working. I'll try some more things and come back with the results later.

**Comment by InnerBushman** [comment]
Ok, here are the results. The xdg-open was throwing out an error unrelated to EDOMH itself but rather related to the url handling: `gio: edomh://capi/?code=(...)&state=secret(...): The specified location is not supported`. After manually re-registering the edomh:// handler again and refreshing cache, all the browsers started to open the edomh:// links correctly (almost).

Here are some of the things I did:
- tested the desktop file manually from `~/.local/share/applications/` - it wasn't working so I added single quotes around the path/name in exec line,
- checked desktop file manually again and now it works, but still not from browser,
- cleared the local mime cache,
- re-registered the handler with: `xdg-mime default your-handler.desktop x-scheme-handler/edomh`,
- updated mime cache with: `update-desktop-database ~/.local/share/applications/`,
- restarted all the browsers,
- opened the app from bin folder, clicked the Connect CMDR button, logged in and got prompted to open EDOMH, 

After confirmation EDOMH correctly received the URL and showed the popup that the account was connected.
I've also tried opening the URL generated by EDSY. Chromium works just fine. In Firefox, on the other hand, a new tab is open but the url is not. I have to manually "go" to the url present in address bar, but I can live with it. 

I'm closing the issue. Cheers!
